cli/cmd/generate.go

```diff
@@ -94,10 +112,38 @@ func genDestination(cmd *cobra.Command, path string, pm *plugin.PluginManager, r
 		return errors.Wrap(err, "failed to get plugin client")
 	}
 	defer destPlugin.Close()
-	res, err := destPlugin.GetClient().GetExampleConfig(cmd.Context())
+	client := destPlugin.GetClient()
+	name, err := client.Name(cmd.Context())
+	if err != nil {
+		return errors.Wrap(err, "failed to get plugin name")
+	}
+	cfg, err := client.GetExampleConfig(cmd.Context())
 	if err != nil {
 		return errors.Wrap(err, "failed to get example config")
 	}
-	fmt.Println(res)
+
+	configPath := name + ".yml"
```

Comment:
let's just ask the user to pass something ending with `yml` otherwise it will be `.yml.yml`

Reply:
This is for when the user hasn't passed in the flag (the name comes from the plugin RPC call)